services/vaadin-directory/vaadin-directory.service.js

```diff
@@ -12,7 +12,128 @@ const {
   age: ageColor,
 } = require('../../lib/color-formatters')
 
-module.exports = class VaadinDirectory extends LegacyService {
+class VaadinDirectoryRating extends LegacyService {
+  static get category() {
+    return 'rating'
+  }
+
+  static get route() {
+    return {
+      base: 'vaadin-directory',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Vaadin Directory',
+        previewUrl: 'rating/vaadinvaadin-grid',
+        keywords: ['vaadin-directory', 'vaadin directory', 'rating'],
+      },
+      {
+        title: 'Vaadin Directory',
+        previewUrl: 'stars/vaadinvaadin-grid',
+        keywords: ['vaadin-directory', 'vaadin directory', 'star', 'stars'],
+      },
+      {
+        title: 'Vaadin Directory',
+        previewUrl: 'rating-count/vaadinvaadin-grid',
+        keywords: [
+          'vaadin-directory',
+          'vaadin directory',
+          'rating-count',
+          'rating count',
+        ],
+      },
+    ]
+  }
+
+  static registerLegacyRouteHandler() {}
+}
+
+class VaadinDirectoryVersion extends LegacyService {
+  static get category() {
+    return 'version'
+  }
+
+  static get route() {
+    return {
+      base: 'vaadin-directory/v',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Vaadin Directory',
+        previewUrl: '/vaadin-directory/v/vaadinvaadin-grid',
```

Comment:
this one is rendering as `/vaadin-directory/v//vaadin-directory/v/vaadinvaadin-grid.svg`